docs/guides/tutorials/getting_started/initialize_a_data_context.rst

```diff
@@ -1,9 +1,9 @@
 .. _tutorials__getting_started__initialize_a_data_context:
 
-Initialize a Data Context
+Set up the tutorial data and initialize a Data Context
 ===============================================
 
-In Great Expectations, your :ref:`Data Context` manages boilerplate configuration. Using a Data Context is almost always the fastest way to get up and running, even though some teams don't need every component of a Data Context.
+In Great Expectations, your :ref:`Data Context` manages your project configuration. Using a Data Context is almost always the fastest way to get up and running, even though some teams don't need every component of a Data Context.
```

Comment:
I strongly prefer the new "your project" wording, and would make the same change in the overview description of this step in the getting started (overview) doc.